syntax = "proto3";

package HyperNovaLabs.Api.Protos.Services;

import "google/api/annotations.proto";
import "google/protobuf/empty.proto";
import "model/Bill.proto";

service ExpenseReportService
{
    rpc CreateBill(CreateBillResquest) returns (CreateBillResponse)
    {
        option (google.api.http) = {
            post: "/api/bill"
            body: "bill"
        };
	}
}

message CreateBillResquest
{
    HyperNovaLabs.Api.Protos.Models.Bill bill = 1;
}

message CreateBillResponse
{
    optional HyperNovaLabs.Api.Protos.Models.Bill bill      = 1;
    bool                                          has_error = 2;
    repeated string                               errors    = 3;
}
